.travis.yml

```diff
@@ -9,3 +9,9 @@ install:
 script:
   - pre-commit run --all-files --verbose
   - mvn clean verify
+  - mvn clean package
```

Comment:
You can just combine `package` into the previous line, no?

Reply:
I wasn't sure 🤷‍♂️ so I just erred on the side of caution. So like this?

```bash
  - mvn clean verify package
```

Reply:
I think `verify` already runs `package` but you'd have to test to be sure.